tests/test_morphio.py

```diff
@@ -50,6 +50,20 @@ def isfloat(s):
     return False
 
 
+def compare_numeric(file1, file2):
```

Comment:
this should probably have a name like `are_same(file1, file2)` since it returns True if they are the same and False if they are not.  Something like that.